### DISCUSSION TITLE
Running server on android

### DISCUSSION THREAD

**Discussion by abdulhakam** [photoprism/photoprism#3018] [Q&A] [answered]
I have 6 or 7 Android phones at home and they are very old, unrootable and they don't support termux otherwise I would install docker in it and use it to run the servers now I want to run the server on them, I am currently using them as FTP servers but I want to use them as photoprism server. Is there an APK package that will allow me to do that? Oldest one has android 4.4 KitKat.

**Comment by graciousgrey** [comment, marked as answer]
We do not yet provide apk packages. Also I am unsure if a phone would meet the [system requirements](https://docs.photoprism.app/getting-started/#system-requirements)?

**Reply by abdulhakam** [reply]
can the server run on a phone with 3GB RAM, 32GB internal memory(at least 21GB available) and octacore arm64 v8. I might have been able to install the server if it was rootable(via termux), but unfortunately, its kind of unique Japanese phone with too much security against system tinkering (it can not even run termux in non-root).

**Reply by lastzero** [reply]
How much RAM? The 32 GB are storage. If it runs, it might be be slow, so not sure if it's worth the effort.

**Reply by abdulhakam** [reply]
thanks, I think I should keep using FTP server for the time being since I have not crossed the 2000 files mark, and I should start organizing the data folder wise now!
